src/audits/AriaRoleNotScoped.js

```diff
@@ -54,7 +54,7 @@ axs.AuditRules.addRule({
         }
         // If we made it this far then no DOM ancestor has a required scope role.
         // Now we need to check if anything aria-owns this element.
-        var owners = axs.utils.getIdReferrers('aria-owns', element);  // there can only be ONE explicit owner but that's a different test
+        var owners = axs.utils.getAriaIdReferrers(element, 'aria-owns');  // there can only be ONE explicit owner but that's a different test
```

Comment:
I changed the API:
1. `axs.utils.getIdReferrers` now returns both HTML and ARIA IDREFs
2. The old "just return ARIA IDREFs" function is renamed to `getAriaIdReferrers`
3. I had to reverse the arguments because arg `attributeName` is now optional `opt_attributeName` - I'm operating on the practice of always putting optional arguments last. We could leave it the other way and pass it `null` if you prefer.